GoogleTestAdapter/Core.Tests/Runners/CommandLineGeneratorTests.cs

```diff
@@ -309,6 +309,27 @@ public void GetCommandLines_ManyTestsWithSuites_BreaksUpLongCommandLinesCorrectl
             splittedTestsAsSet.Should().BeEquivalentTo(testsAsSet);
         }
 
+    [TestMethod]
+    [TestCategory(Unit)]
+    public void GetCommandLines_HugeNumberOfSuitesToBeExecutedFully_WontBreakCommandLineCreation()
+    {
+      var range = Enumerable.Range(0, 400);
+
+      var allTests = range.Select(idx => "MyTestSuiteWithSomeWhatLongNameIsStillHandledCorrectly" + idx + ".Test")
+             .Concat(range.Select(idx => "MyTestSuiteWithSomeWhatLongNameIsStillHandledCorrectly" + idx + ".Test2"))
+             .Concat(range.Take(20).Select(idx => "MyTestSuiteWithSomeWhatLongNameIsStillHandledCorrectly" + idx + ".Test3"))
+             .ToList();
+
+      var testsToExecute = allTests.Where((testcase, idx) => idx < 700 || idx % 17 == 0).ToList();
+
+      IEnumerable<Model.TestCase> testCases = TestDataCreator.CreateDummyTestCasesFull(testsToExecute.ToArray(), allTests.ToArray());
+
+      List<CommandLineGenerator.Args> commands = new CommandLineGenerator(testCases, TestDataCreator.DummyExecutable.Length, "", "", TestEnvironment.Options)
```

Comment:
Since the xml file parameter will never be empty as far as I recall, it's probably better to pass a non-empty string for that parameter.

Reply:
I just did copy & paste from one of the existing test ... you're the master here ;)

Reply:
I was pretty sure that I would complain about stuff that I had done myself (and which you had used as base of your code) :-) :-)

Reply:
:D - isn't is terrible that if you come back to your code half a year later you sometimes think - who implemented this crap ;)